examples/apps/djapp/legacy/4_create_initial_mesh_components/nodes_representing_virtual_services.yaml

```diff
@@ -13,7 +13,7 @@ spec:
     dns:
       hostName: metal.prod.svc.cluster.local
```

Comment:
`hostName` -> `hostname`  

Needs to be changed in all the places

Reply:
This was cruft leftover from notes I made early on. Fixed. Everything in legacy is the old version of the app.